admin/pb_migrations/1708362089_updated_templates.js

```diff
@@ -0,0 +1,46 @@
+/// <reference path="../pb_data/types.d.ts" />
+migrate((db) => {
+  const dao = new Dao(db)
+  const collection = dao.findCollectionByNameOrId("7amth7pix3i8k5p")
+
+  // remove
+  collection.schema.removeField("xozvdofz")
```

Comment:
Should we normalize these migrations?

Reply:
maybe with a little script?

Reply:
Also here, we should make an issue on starters about this